### PR TITLE
.github/pr-doclint.yaml: add document lint scan CI

### PR DESCRIPTION
**Pull request by kenplusplus**

### CODE REVIEW

File: .github/cspell.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        ".git/"
+    ],
+    "ignorewords": [],
+    "import": [],
+    "dictionarydefinitions": [],
+    "dictionaries": [],
+    "words": [
+        "acpi",
+        "ccnp",
+        "ccel",
```
Comment (leyao-daily) [inline .github/cspell.json]: Do we need sort them alphabetically? switch ccap <-> ccel and srtm <-> setuptools.

Comment (kenplusplus) [reply .github/cspell.json]: how about only keep "cnap" in the list firstly, please see new commit

Comment (leyao-daily) [reply .github/cspell.json]: It makes sense, add new words if needed.

**Review by leyao-daily — APPROVED** [review]
LGTM.

**Pull request merged**